Dockerfile

```
@@ -43,34 +43,46 @@
-RUN mkdir -p /app/etc && \
-    echo "import Config" > /app/etc/wanderer_notifier.exs
+RUN echo "import Config" > /app/etc/wanderer_notifier.exs
 
 # Copy the release from the builder
 COPY --from=builder /app/_build/prod/rel/wanderer_notifier ./
 
+# Copy runtime scripts
+COPY scripts/start_with_db.sh /app/bin/
+COPY scripts/db_operations.sh /app/bin/
+RUN chmod +x /app/bin/start_with_db.sh
+RUN chmod +x /app/bin/db_operations.sh
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)